SCClassLibrary/SCDoc/SCDoc.sc

```diff
@@ -549,7 +549,11 @@ SCDoc {
 					};
 				}
 			};
-			rootPaths = [thisProcess.platform.userExtensionDir, thisProcess.platform.systemExtensionDir];
+			rootPaths = if(LanguageConfig.excludeDefaultPaths) {
```

Comment:
excuse my ignorance -- is this flag set if `sclang` is run with `-a` but the config file itself doesn't contain "excludeDefaultPaths: true"? (@miguel-negrao ?)

Reply:
I can confirm it does: https://github.com/miguel-negrao/supercollider/blob/208f56338871bcd2970585b76db9da9cf834e3cc/lang/LangSource/SC_LanguageConfig.cpp#L148

Thanks for doubting on this (I mean it!), when you asked I wasn't 100% sure anymore either. I had tested with -a but then I noticed that build_sclang.cfg (which I was using while making this PR) has `excludeDefaultPaths: true` too. Anyway, I also double-checked by removing that option from config file, and it works. It also sets the flag to false if the option is not present in config and -a is not used.

Reply:
thanks so much!